twopence/run

```diff
@@ -366,9 +366,13 @@ def nfs_write_exports(server, exports):
 		journal.failure("unable to upload exports file")
 		return False
 
-	if nfs_server_control(server, "reload"):
-		return False
-
+	if (reload_nfs):
+		if nfs_server_control(server, "reload"):
+			return False
+	else:
+		journal.info("--- skipping reload of nfs configuration ---")
+		journal.success(" export file updated")
+		return True
```

Comment:
Use bolean and skip the additional flush tests, since nfs server is stopped
---------------------------------
TEST: setup: write simple exports file to server
server: updating exports file:
/srv/nfs/dir1 *(rw)
---end of exports file---
server: uploading /etc/exports
--- skipping reload of nfs configuration ---
 export file updated
SUCCESS